daily-patch-information/src/main/java/org/wso2/patchinformation/email/EmailContentCreator.java

```diff
@@ -0,0 +1,268 @@
+//
+// Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+package org.wso2.patchinformation.email;
+
+import org.wso2.patchinformation.comparators.DaysInStateComparator;
+import org.wso2.patchinformation.comparators.JIRADateComparator;
+import org.wso2.patchinformation.comparators.ReleasedReportDateComparator;
+import org.wso2.patchinformation.comparators.ReportDateComparator;
+import org.wso2.patchinformation.constants.Constants;
+import org.wso2.patchinformation.jira.JIRAIssue;
+import org.wso2.patchinformation.pmt.InactivePatch;
+import org.wso2.patchinformation.pmt.OpenPatch;
+import org.wso2.patchinformation.pmt.Patch;
+
+import java.util.ArrayList;
+
+import static org.wso2.patchinformation.constants.EmailConstants.COLUMN_NAMES;
+import static org.wso2.patchinformation.constants.EmailConstants.COLUMN_NAMES_DEV;
+import static org.wso2.patchinformation.constants.EmailConstants.COLUMN_NAMES_INACTIVE;
+import static org.wso2.patchinformation.constants.EmailConstants.COLUMN_NAMES_RELEASED;
+import static org.wso2.patchinformation.constants.EmailConstants.COLUMN_NAMES_SUMMARY;
+import static org.wso2.patchinformation.constants.EmailConstants.EMAIL_FOOTER;
+import static org.wso2.patchinformation.constants.EmailConstants.SECTION_HEADER_DEV;
+import static org.wso2.patchinformation.constants.EmailConstants.SECTION_HEADER_INACTIVE;
+import static org.wso2.patchinformation.constants.EmailConstants.SECTION_HEADER_RELEASED;
+import static org.wso2.patchinformation.constants.EmailConstants.SECTION_HEADER_SIGNING;
+import static org.wso2.patchinformation.constants.EmailConstants.SECTION_HEADER_SUMMARY;
+
+/**
+ * Creates and returns the body of the email.
+ */
+public class EmailContentCreator {
+
+    private static EmailContentCreator emailContentCreator;
+
+    private EmailContentCreator() {
+    }
+
+    public static EmailContentCreator getEmailContentCreator() {
+        if (emailContentCreator == null) {
```

Comment:
You can instantiate the new instance of the emailContentCreator on line#49, therefore you don't need to check for null, and it will be simply a getter.